### PR TITLE
Fix some small typos

### PR DESCRIPTION
**Pull request by jakejarvis**
That's all! 😊

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,14 +3,16 @@
...
 
 ## Keep in mind the following:
-- This repository was created using a JWT [token generated](https://github.com/bdougie/open-sauced/blob/master/src/lib/identityActions.js) when you log into [opensauced.pizza](https://opensauced.pizza). 
-- Deleting this repo, will pernamently delete your [opensauced.pizza](https://opensauced.pizza) data, but will not remove access to your GitHub account data.You will need delete Open Sauced's access to your GitHub account by visiting your GitHub profiles [integration access settings](https://github.com/settings/apps/authorizations).
+
+- This repository was created using a JWT [token generated](https://github.com/bdougie/open-sauced/blob/master/src/lib/identityActions.js) when you logged into [opensauced.pizza](https://opensauced.pizza). 
+- Deleting this repo will permanently delete your [opensauced.pizza](https://opensauced.pizza) data, but will not remove access to your GitHub account data. You will need delete Open Sauced's access to your GitHub account by visiting your GitHub profile's [integration access settings](https://github.com/settings/apps/authorizations).
```
Comment (yg) [inline README.md:16]:
```suggestion
- Deleting this repo will permanently delete your [opensauced.pizza](https://opensauced.pizza) data, but will not remove access to your GitHub account data. You will need to delete Open Sauced's access to your GitHub account by visiting your GitHub profile's [integration access settings](https://github.com/settings/apps/authorizations).
```

**Comment by yg** [comment]
Thanks @jakejarvis!

:purple_heart:

**Comment by bdougie** [comment]
Thanks @jakejarvis, stay saucey.

**Pull request merged**